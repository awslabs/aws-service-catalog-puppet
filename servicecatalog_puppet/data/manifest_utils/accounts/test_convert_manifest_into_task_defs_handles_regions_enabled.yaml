schema: puppet-2019-04-01
accounts:
  - account_id: '246919575282'
    default_region: eu-west-1
    name: '246919575282'
    regions_enabled:
      - eu-west-1
      - eu-west-2
    tags:
      - type:prod
      - partition:eu
      - scope:pci
launches:
  assumable-role-account:
    portfolio: example-simple-central-it-team-portfolio
    product: assumable-role-account
    version: v1
    parameters:
      RoleName:
        default: AccountAssumeRole
      Path:
        default: /
      AccountToTrust:
        default: '246919575282'
    deploy_to:
      accounts:
        - account_id: '246919575282'
          regions: regions_enabled
    outputs:
      ssm:
        - param_name: ServiceCatalogAccountRoleArn
          stack_output: RoleArn