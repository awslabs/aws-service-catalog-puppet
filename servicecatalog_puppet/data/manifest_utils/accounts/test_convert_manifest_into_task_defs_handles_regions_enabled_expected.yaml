- account_id: '246919575282'
  account_parameters: {}
  dependencies: []
  launch_name: assumable-role-account
  launch_parameters:
    AccountToTrust:
      default: '246919575282'
    Path:
      default: /
    RoleName:
      default: AccountAssumeRole
  manifest_parameters: {}
  parameters: []
  portfolio: example-simple-central-it-team-portfolio
  product: assumable-role-account
  puppet_account_id: 9
  region: eu-west-1
  requested_priority: 0
  retry_count: 0
  should_use_sns: true
  should_use_product_plans: true
  ssm_param_inputs: []
  pre_actions: []
  post_actions: []
  ssm_param_outputs:
  - param_name: ServiceCatalogAccountRoleArn
    stack_output: RoleArn
  status: provisioned
  version: v1
  worker_timeout: 0
- account_id: '246919575282'
  account_parameters: {}
  dependencies: []
  launch_name: assumable-role-account
  launch_parameters:
    AccountToTrust:
      default: '246919575282'
    Path:
      default: /
    RoleName:
      default: AccountAssumeRole
  manifest_parameters: {}
  parameters: []
  portfolio: example-simple-central-it-team-portfolio
  product: assumable-role-account
  puppet_account_id: 9
  region: eu-west-2
  requested_priority: 0
  retry_count: 0
  should_use_sns: true
  should_use_product_plans: true
  ssm_param_inputs: []
  pre_actions: []
  post_actions: []
  ssm_param_outputs:
  - param_name: ServiceCatalogAccountRoleArn
    stack_output: RoleArn
  status: provisioned
  version: v1
  worker_timeout: 0  